back-end/src/Plant/plant.controller.js

```diff
@@ -0,0 +1,17 @@
+const {Plant} = require('./plant.model')
+
+exports.getPlant = async (req,res) => {
+    try {
+        res.status(200).send(req.plant)
+    } catch (error) {
+        res.status(500).send({message:"Plant not found"})
```

Comment:
Use the json function if you're sending back json data

Reply:
do you mean res.status(200).json.(req.plant)